addons/website_forum/static/src/js/website_forum.js

```diff
@@ -532,18 +533,22 @@ publicWidget.registry.websiteForum = publicWidget.Widget.extend({
      */
     _onDeleteCommentClick: function (ev) {
         ev.preventDefault();
-        var $link = $(ev.currentTarget);
-        var $container = $link.closest('.o_wforum_post_comments_container');
-
-        rpc($link.closest('form').attr('action')).then(function () {
-            $link.closest('.o_wforum_post_comment').remove();
-
-            var count = $container.find('.o_wforum_post_comment').length;
-            if (count) {
-                $container.find('.o_wforum_comments_count').text(count);
-            } else {
-                $container.find('.o_wforum_comments_count_header').remove();
-            }
+        this.call("dialog", "add", ConfirmationDialog, {
+            body: _t("Are you sure you want to delete this comment?"),
+            confirmLabel: _t("Delete"),
+            confirm: () => {
+                const deleteBtn = ev.currentTarget;
+                rpc(deleteBtn.closest("form").attributes.action.value).then(() => {
```

Comment:
Ah yes I remember. Can you recall me what the action was? :)

Reply:
The action? :thinking: What do you mean?

Adding some context hoping I answer your question :smile: : 

It calls the route to delete a comment here :slightly_smiling_face: 
I improved the code a bit comparing to last time by adding a catch to the rpc as the python can throw an exception :slightly_smiling_face: And removing the is_admin from the xml as explained in the commit message :smile: 
(As a reminder, the task we were waiting has fixed the fact that the warning alert was displaying the post deletion karma instead of the comment deletion karma)

Reply:
Ah yes my bad. I just had to read the link_button template to remember missing details. Thanks for the context.